Cargo.toml

```diff
@@ -8,6 +8,7 @@ rust-version = "1.72"
 version = "0.1.0"
 
 [workspace.dependencies]
+serde_json = { version = "1.0.114", features = ["float_roundtrip"] }
```

Comment:
`float_roundtrip` is a needed feature to preserve the precision of floats when serializing and deserializing.  Without this there were slight precision errors in the invertibility test.